packages/type-utils/tests/isTypeReadonly.test.ts

```diff
@@ -137,6 +137,34 @@ describe('isTypeReadonly', () => {
           );
         });
       });
+      
+      describe('Conditional Types', () => {
```

Comment:
Something has gone wrong somewhere along the way. These are the tests for #4421

Reply:
did i copy pasta the wrong merge?
oops!

Reply:
sorry - feel free to revert my merge and re-merge main properly!

feel free to rebase your other 3 PRs on top of each other ahead of time so the main merges go smoother - otherwise they'll conflict each time I believe.

Reply:
Will do. I set them up to merge in order of there PR number